DirectProgramming/C++/CombinationalLogic/MandelbrotOMP/README.md

```diff
@@ -50,12 +50,15 @@ The basic steps to build and run a sample using VS Code include:
  - Configure the oneAPI environment with the extension **Environment Configurator for Intel oneAPI Toolkits**.
  - Open a Terminal in VS Code (**Terminal>New Terminal**).
  - Run the sample in the VS Code terminal using the instructions below.
+ - (Linux only) Debug your GPU application with GDB for Intel®
+oneAPI toolkits using the **Generate Launch Configurations** extension.
 
-To learn more about the extensions and how to configure the oneAPI environment, see
+To learn more about the extensions, see
```

Comment:
@tomlenth I removed "and set environment variables" after "To learn more about the extensions" because now we are talking about the environment variable extension and the debug extension (launch configurator).